presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperator.java

```diff
@@ -510,6 +511,19 @@ public void close()
         closed = true;
         probe = null;
 
+        if (partitionedConsumption == null) {
```

Comment:
I would add a comment here that this code is needed for the case of early termination where the operator is closed before being finished.